articles/cognitive-services/Computer-vision/computer-vision-resource-container-config.md

```diff
@@ -156,6 +156,6 @@ You can specify an input mount or output mount by specifying the `--mount` optio
 
 For example, the following command defines a Docker bind mount to the `D:\Output` folder on the host machine as the output mount, then instantiates a container from the Recognize Text container image, saving log files in JSON format to the output mount.
 
-  ```Docker
```

Comment:
note: this still probably isn't a correct language block because this is actually a block of commands